app/src/main/java/com/sdp13epfl2021/projmag/database/impl/firebase/FirebaseUserdataDatabase.kt

```diff
@@ -258,8 +257,7 @@ class FirebaseUserdataDatabase @Inject constructor(
     ) {
         firestore
             .collection(USER_PROFILE)
-            .document(userID)
-            .get()
+            .document(userID).get()
             .addOnSuccessListener { document ->
```

Comment:
Isn't it more readable to keep the .get() on another line ?

Reply:
yes it was to get rid of a code climate warning, I can put it back if you like